### ISSUE TITLE
Late Agena B / Agena D missing ullage RCS thrusters

### ISSUE THREAD

**Issue by CAPFlyer** [CobaltWolf/Bluedog-Design-Bureau#1168]
I've noted that the Agena B & Agena D equipment racks lack provision for RCS-based ullage thrusters.  Per available documentation, the later Agena B and all Agena D had RCS-based ullage thrusters instead of the solid rocket-based ones.

See - 
http://www.b14643.de/Spacerockets_2/United_States_4/Agena/index.htm (Section called "Agena B, D, & D+)

Agena B with integrated Ullage Thrusters/Rockets  - http://www.b14643.de/Spacerockets_2/United_States_4/Agena/B_3.jpg

Agena D with integrated Ullage Thursters/Rockets - http://www.b14643.de/Spacerockets_2/United_States_4/Agena/D_1.jpg

I also found this which mentions "secondary propulsion" which I would take to be the Ullage Thrusters -

https://forum.nasaspaceflight.com/index.php?action=dlattach;topic=16154.0;attach=315794;image

**Comment by Kosmos172** [comment]
> I also found this which mentions "secondary propulsion" which I would take to be the Ullage Thrusters -

The secondary propulsion system was a thruster package attached to the sides of the Agena D aft rack and is included in the mod. You can see the packages either side of the main engine in this image:
https://upload.wikimedia.org/wikipedia/commons/2/2f/Gemini_and_Agena_D.jpg

**Comment by CAPFlyer** [comment]
Yes, but the reference image (had you looked at it) said for AGENA B under "Ancilliary Devices".  Agena B had a different equipment rack.

**Comment by CobaltWolf** [comment]
I can actually see that little green thruster being a nice part to have. I'll see if I can bash something together some time!

**Comment by Pappystein** [comment]
Agena D did not need Ullage.    The whole point of the upgrade from the B to the D was twofold.  A) Digital electronics replaced analog clock-based electronics saving weight and length (approximately 4 inches) and B) The new sump system allowed for zero Ullage in low/no gravity.    

This is all called out here:
https://ntrs.nasa.gov/search?q=19720013179

The document is actually focused on the Ascent Agena but it coroberates many other agena documents all in one simple pdf.

**Comment by CAPFlyer** [comment]
Sorry to be so long responding to this.  I don't agree with your assertion Pappy.  The document you linked doesn't make a blanket statement like you did.  In point, there is this from Page 1-4 - "...and devices were implemented as each major block of vehicles was manu-factured," and this lists the development of the advanced ullage control *after* that statement, suggesting it was an improvement created within Agena D production, not as part of the Agena B -> Agena D improvements.  My links suggest this as well, that the ullage improvement was something introduced in the mid/late Agena D production run, and thus the need for the small thruster configuration seen on many of the early Agena D launches and most especially on the Reconnaissance and Intelligence flights.
